internal/server/authn/method/github/server.go

```diff
@@ -28,6 +28,7 @@ const (
 	githubAPI                        = "https://api.github.com"
 	githubUser              endpoint = "/user"
 	githubUserOrganizations endpoint = "/user/orgs"
+	githubUserTeams         endpoint = "/user/teams?per_page=100"
```

Comment:
I take it we're just going to assume likelihood is 99.9% of folks have less that 100 team memberships and cross that bridge when we come to it for the outliers?

Seems reasonable to me, just clarifying.

Reply:
Exactly, I can't image someone having more than 100 team memberships (even they may exists 😆)!

Also, this is the same number used by Grafana as @markphelps pointed out: [SocialGithub.fetchTeamMemberships](https://github.com/grafana/grafana/blob/c0933fa6bb4038c519466d12bd2e61693e32a4e1/pkg/login/social/connectors/github_oauth.go#L207)

Reply:
That's awesome. Makes total sense.